### ISSUE TITLE
Newly Merged tBTC Bridge showing data from another contract

### ISSUE THREAD

**Issue by Luna5-threshold** [DefiLlama/DefiLlama-Adapters#5346]
Good day!

This PR was created Feb 13th: [https://github.com/DefiLlama/DefiLlama-Adapters/pull/5319](https://github.com/DefiLlama/DefiLlama-Adapters/pull/5319)
And was merged Feb 15th. 

It contained all the information regarding the tBTC Bridge, including logo, ticker, audits, contract address, etc. 

We now find that it is showing: 
* The wrong ticker 
* The wrong contract address
* It says it is not audited. 
* Wrong CoinGecko space

Please this is the correct information (already provided in the original PR): 

- **Audits**: **Least Authority** https://leastauthority.com/blog/audits/audit-of-keep-network-tbtc-bridge-v2/
- **Coingecko**: https://www.coingecko.com/en/coins/tbtc
- **Ticker**: tBTC
- **Contract address**: https://etherscan.io/token/0x18084fba666a33d37592fa2633fd49a74dd93a88

Please note that -> _The adapter is working properly, because it was implemented using the correct contract._ 

As a clarification again, tBTC version 2 belongs to the Threshold Network project, and it is one of the products provided by Threshold Network. However, it is **different** from our work token (T). You are currently displaying information on the T token, used for governance and staking, but showing data from the adapter built for the tBTC token (see image). 

Please let me know if I can help solving any further doubts or questions. 

![defillama-tBTC-wrong](https://user-images.githubusercontent.com/110028834/219341027-cc30c2c1-5def-4d43-ab6d-0dc74b5f0650.png)

**Comment by Luna5-threshold** [comment]
@realdealshaman Please can you review this issue? 
We have this problem related to PR https://github.com/DefiLlama/DefiLlama-Adapters/pull/5319

**Comment by Define101** [comment]
Sorry for the late reply
Everything was updated. Should be live within 1 hour

**Comment by Define101** [comment]
Hello, left a comment on the discord will leave it here too
Sorry for this situation

"Hello East. Regarding these changes, we will need to revert it. At first, we added the T token because our rule is to always add the governance token. After some internal conversations, we decided to stick with this rule, for that reason, the contract token will be T since it is the governance token from Threshold. Another similar example where we use the exact same methodology is with Interlay. Interlay BTC token address is the governance token. I am sorry for this confusion, it was a mistake on my end. Hope you understand, any questions feel free to let us know. (the audit is still there ofc)"
